docs/index.md

```diff
@@ -1,37 +1,37 @@
 # Your Universal Cellular Automata
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/riveSunder/yuca/master/assets/s11_slow_glider_vagopteron_glider.gif">
+<img src="https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/s11_slow_glider_vagopteron_glider.gif">
   <br><br><strong>Left:</strong> Evolved slow glider from s11 Glaberish CA rule set. <strong>Right:</strong> evolved glider in <em>Vagopteron</em> Lenia rule set. 
 </p>
 
 ## A continuous cellular automata research library
 
 
-####  Davis, Q, Tyrell and Bongard, Josh. [Selecting Continuous Life-Like Cellular Automata for Halting Unpredictability: Evolving for Abiogenesis](https://rivesunder.github.io/yuca/evo_abio). Accepted to Proceedings of the Genetic and Evolutionary Computation Conference Companion (2022).  
+####  Davis, Q, Tyrell and Bongard, Josh. [Selecting Continuous Life-Like Cellular Automata for Halting Unpredictability: Evolving for Abiogenesis](https://rivesunder.github.io/yuca_docs/evo_abio). Accepted to Proceedings of the Genetic and Evolutionary Computation Conference Companion (2022).  
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.